application/frontend/template/template.html

```diff
@@ -0,0 +1,76 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Your Web Page</title>
+
+    <!-- Bootstrap CSS -->
+    <link
+      rel="stylesheet"
+      href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css"
+    />
+    <!-- CSS stylesheet -->
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <!-- Wrapping Div -->
+    <div class="wrapper">
+      <!-- Navbar -->
+      <nav class="navbar navbar-expand-lg navbar-dark bg-dark">
+        <div class="container">
+          <a class="navbar-brand" href="#">
+            <img
+              class="brand-logo"
+              src="../assets/images/watchdogs.png"
+              alt="brand-logo"
+            />
+          </a>
+          <button
+            class="navbar-toggler"
+            type="button"
+            data-toggle="collapse"
+            data-target="#navbarNav"
+            aria-controls="navbarNav"
+            aria-expanded="false"
+            aria-label="Toggle navigation"
+          >
+            <span class="navbar-toggler-icon"></span>
+          </button>
+          <div class="collapse navbar-collapse" id="navbarNav">
+            <ul class="navbar-nav ml-auto">
+              <li class="nav-item">
+                <a class="nav-link" href="#">Customer</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="#">Employee</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="#">About</a>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </nav>
+
+      <!-- Page Content -->
+      <div class="container">
+        <h1>Main Content to go here</h1>
+        <p>
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Ut iure enim
+          necessitatibus, voluptatum temporibus illo veniam totam quidem ratione
+          fugit recusandae adipisci iste blanditiis optio praesentium asperiores
+          a fugiat quisquam.
+        </p>
+      </div>
+      <footer class="footer">
+        <div class="container">
+          <p>&copy; 2023 Code Crafters. Made with ❤️.</p>
+        </div>
+      </footer>
+    </div>
+    <script src="https://code.jquery.com/jquery-3.5.1.slim.min.js"></script>
+    <script src="https://cdn.jsdelivr.net/npm/@popperjs/core@2.5.3/dist/umd/popper.min.js"></script>
```

Comment:
do we require these two external libs, read what Pavan Sir has to say.

Reply:
yes for collapsable navbar